tests/fpsemi-examples.cpp

```diff
@@ -1106,7 +1226,8 @@ namespace libsemigroups {
   }
 
   // Chinese monoid
-  // See: The Chinese Monoid - Cassaigne, Espie, Krob, Novelli and Hivert, 2001
+  // See: The Chinese Monoid - Cassaigne, Espie, Krob, Novelli and Hivert,
+  // 2001
```

Comment:
This is an accidental change from where the formatter alters a file I've modified, but in a place where I've not modified it. I can remove this if that's best.

Reply:
Yes please.

Reply:
Done